check_prerequisites.sh

```diff
@@ -167,7 +168,7 @@ fi
 
 echo -e "${BLUE}Checking health status of CAS '$CAS' in namespace '$CAS_NAMESPACE' is provisioned${NC}"
 
-if ! kubectl provision cas "$CAS" -n "$CAS_NAMESPACE" --is-provisioned
+if ! VERSION="" kubectl provision cas "$CAS" -n "$CAS_NAMESPACE" --is-provisioned
```

Comment:
Why not `if ! kubectl provision cas "$CAS" -n "$CAS_NAMESPACE" --is-provisioned --set-version ${VERSION}`?